gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
-distributionBase = GRADLE_USER_HOME
-distributionPath = wrapper/dists
-distributionUrl = https://services.gradle.org/distributions/gradle-7.6-bin.zip
-zipStoreBase = GRADLE_USER_HOME
-zipStorePath = wrapper/dists
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.1.1-bin.zip
```

Comment:
decided not to ship fork until we have full auth support and other components (otherwise we'd have to stop dogfooding build caching)